README.rst

```diff
@@ -1,148 +1,262 @@
+This project is part of FIWARE_ and `FIWARE Ops`_ suite of tools for the operation of FIWARE Lab.
+
+Any feedback on this documentation is highly welcome, including bugs, typos or
+things you think should be included but aren't. You can use `github issues`__
+to provide feedback.
+
+__ `Fiware-facts - GitHub issues`_
+
+GEi overall description
+=======================
+Policy Manager is called Bosun.
+
+Bosun GEri offers decision-making ability, independently of the type of resource (physical/virtual resources,
+network, service, etc.)  being able to solve complex problems within the Cloud field by reasoning about the knowledge
+base, represented by facts and rules.
+Bosun GEri provides the basic management of cloud resources based on rules, as well as management of the corresponding
+resources within FIWARE Cloud instances based on infrastructure physical monitoring, resources and services
+security monitoring or whatever that could be defined by facts, actions and rules.
+
+The baseline for the Bosun GEri is PyCLIPS, which is a module to interact with CLIPS expert system implemented in
+python language. The reason to take PyCLIPS is to extend the OpenStack ecosystem with an expert system, written in
+the same language as the rest of the OpenStack services.
+Besides, It provides notification service to your own HTTP server where you can define your
+own actions based on the notifications launched by Policy Manager.
+Last but not least, Bosun is integrated with the Monitoring GEri in order to recover the information of the (virtual)
+system and calculate any possible change on it based on the knowledge database defined for it.
+
+Components
+----------
+
+Fiware-Cloto
+    Fiware-cloto is part of FIWARE Policy Manager. It provides an API-REST to create rules associated to servers,
+    subscribe servers to Context Broker to get information about resources consumption of that servers and launch actions
+    described in rules when conditions are given.
+
+Fiware-Facts
+    Server to process the incoming facts from the
+    `Orion Context Broker <https://github.com/telefonicaid/fiware-orion>`__
+    and publish the result into a RabbitMQ queue to be analysed by Fiware-Cloto. The facts are the result of the server
+    resources consumption.
+
+For more information, please refer to the `documentation <https://github.com/telefonicaid/fiware-cloto/tree/develop/doc/README.rst>`_.
+
+Build and Install
+=================
+
+Requirements
+------------
+
+- Operating systems: CentOS (RedHat) and Ubuntu (Debian), being CentOS 6.3 the
+  reference operating system.
 
-Prerequisites
-=============
 To install this module you have to install some components:
 
 - Python 2.7
 - Fiware-cloto module (https://github.com/telefonicaid/fiware-cloto)
 - Redis 2.9.1 or above
-- gunicorn 19.1.1 or above
+- RabbitMQ Server 3.3.0 or above (http://www.rabbitmq.com/download.html)
 
 Installation
```

Comment:
Installation from source code should be part of the Installation manual, here I prefer to include only the installation using pypi.